tests/integration/tier0/check-user-response/test_user_response.py

```diff
@@ -52,3 +56,48 @@ def test_check_user_response_organization(convert2rhel):
         # the Ctrl+d is used to terminate the process instead
         c2r.sendcontrol("d")
     assert c2r.exitstatus != 0
+
+
+def test_auto_attach_pool_submgr(convert2rhel):
+    with convert2rhel(
+        "-y --no-rpm-va --serverurl {} --username {} --password {} --debug".format(
+            env.str("RHSM_SERVER_URL"),
+            env.str("RHSM_SINGLE_SUB_USERNAME"),
+            env.str("RHSM_SINGLE_SUB_PASSWORD"),
+        )
+    ) as c2r:
+        c2r.expect(" is the only subscription available, it will automatically be selected for the conversion.")
+        c2r.expect_exact("Registering the system using subscription-manager ...")
+        if "oracle-7" in booted_os or "centos-7" in booted_os:
+            c2r.sendcontrol("c")
+            c2r.sendcontrol("d")
+        else:
+            c2r.sendcontrol("c")
+
+        assert c2r.expect("Rollback: RHSM-related actions") == 0
+
+        assert c2r.exitstatus != 0
```

Comment:
This test is currently blocked because of a problem we have with the account `RHSM_SINGLE_SUB_USERNAME`. Once this is done, we can properly test that feature out. 

## More on the blocker: 

We need an account that only has 1 subscription attached to it, but, in every account, we created with the internal test tool we have, the account got another extra subscription to it. Since this is happening, we can't properly test this feature out and need to wait on this blocker to be fixed.

Currently we have an issue created about this for the internal service that handles this stuff and a ticket open for the IT to see if they can remove that extra subscription from our test account.

Reply:
This issue is not happening anymore since we discovered a way of adding only one subscription to an account. I have tested it locally and this issue seems to be fixed.